tests/callbacks/test_progress_bar.py

```diff
@@ -193,3 +193,50 @@ def on_test_batch_end(self, trainer, pl_module, batch, batch_idx, dataloader_idx
 
     trainer.test(model)
     assert progress_bar.test_batches_seen == progress_bar.total_test_batches
+
+
+@pytest.mark.parametrize('num_sanity_val_steps,num_val_dataloaders_batches', [
+    (-1, [10]),
```

Comment:
I think it would make the test simpler if you added a third argument with the expected number of steps. Then you can simply check the expected value below, and don't have to implement the logic.

Reply:
Thank you. I've fixed it.